### ISSUE TITLE
Mesh generations: keeps only one hemisphere

### ISSUE THREAD

**Issue by FedeClaudi** [brainglobe/bg-atlasgen#4]
The function `mesh_utils.extract_mesh_from_mask` call's vedo's `mesh.extractLargestRegion()` ([here](https://github.com/brainglobe/bg-atlasgen/blob/86caab99317359e5f791c991febb12589c5ced8a/bg_atlasgen/mesh_utils.py#L117)) after using marching cubes on a image stack to generate the mesh.

This is a problem for atlases with two hemispheres as brain regions which are not connected along the mid-line  will generate two large meshes and one will be excluded.  This has been fixed by others by selecting the two largest structures, but it remains to be implemented in the atlas generation code.

**Comment by vigji** [comment]
isn't this fixed with last PR from @MathieuBo ?

**Comment by adamltyson** [comment]
I thought so:
https://github.com/brainglobe/bg-atlasgen/blob/f944baaf69004aca270c29dee50592369969db72/bg_atlasgen/mesh_utils.py#L151-L152

**Comment by MathieuBo** [comment]
Yes it should (at least it did in my case). However, @FedeClaudi comment's was before my PR. Do you still have the issue now?
